src/rustup-cli/self_update.rs

```diff
@@ -660,6 +668,11 @@ fn customize_install(mut opts: InstallOpts) -> Result<InstallOpts> {
         &opts.default_toolchain,
     )?;
 
+    opts.profile = common::question_str(
+        "Profile (which tools and data to install)? (minimal/default/complete)",
```

Comment:
Again it worries me that we've hard-coded the set of profiles in this string too.